AUTHORS

```diff
@@ -4,17 +4,38 @@ partial, alphabetical list of developers and contributors to Cantera over the
 years. If you've been left off, please report the omission on the Github issue
 tracker.
 
+Emil Atz
 Philip Berndt
+Tilman Bremer
 Victor Brunini, Sandia National Laboratory
 Bang-Shiuh Chen, Purdue University
+Ryan Crisanti
+Nicholas Curtis
 Steven DeCaluwe, Colorado School of Mines
+Vishesh Devgan
 Thomas Fiala, Technische Universität München
 David Fronczek
+@g3bk47
+Matteo Giani
 Dave Goodwin, California Institute of Technology
 John Hewson, Sandia National Laboratory
+Trevor Hickey
+Yuanjie Jiang
+JonKrisotfer
+Kyle Linevitch, Jr.
+Christopher Leuth
 Nicholas Malaya, University of Texas at Austin
+Evan McCorkle
+Ivan Mitrichev, Mendeleev University of Chemical Technology of Russia
 Harry Moffat, Sandia National Laboratory
+Christopher Neal
+Kyle Niemeyer, Oregon State University
+Petr (@petrkle)
```

Comment:
Why is this handled differently from all of the others?

Reply:
I ended up removing this one, because the code that was added by this PR on the website was removed because it caused the regression removing the docs etc.